clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/MigrateLoadBalancerStrategySpec.groovy

```diff
@@ -600,4 +601,60 @@ class MigrateLoadBalancerStrategySpec extends Specification {
     1 * amazonEC2.describeVpcs(_) >> new DescribeVpcsResult().withVpcs(new Vpc().withTags(new Tag("Name", "vpc1")))
     1 * sourceLookup.createSecurityGroup({ it.name == 'app-elb'}) >> new SecurityGroupUpdater(new SecurityGroup(groupId: 'sg-elb'), amazonEC2)
   }
+
+  @Unroll
+  void 'only adds Reference-Security-Policy attribute if present when creating policy'() {
```

Comment:
Bit of personal preference but you've got a lot of unnecessary mocks in here for the specific change you're testing.

If you pulled the policy copying out into a dedicated method, you could test it more explicitly without the use of mocks.

```
Collection<PolicyAttributes> clonePolicyAttributes(Policy sourcePolicy) {
...
}
```
(with whatever the appropriate return/parameter types are).